api.rst

```diff
@@ -139,6 +115,49 @@ The view will provide a new endpoint:
 
     curl "http://localhost:3000/fresh_stories"
 
+.. _logical_operators:
+
+Logical operators
+~~~~~~~~~~~~~~~~~
+
+Multiple parameters can be logically **conjoined** using ``and``:
```

Comment:
This paragraph makes me think that to combine filters with `and` I have to use the `and=(..)` notation.

So I think this section should start with something like "by default filters are combined with AND but you can use OR with the following notation..". OR should be the main focus.